fhir-converter/src/templates/cda/ValueSet/MedicationStatementStatus.hbs

```diff
@@ -1,11 +1,14 @@
-{{#if code}}
-  {{#if (eq (toLower code) 'aborted')}}
-    "stopped"
-  {{else if (eq (toLower code) 'ordered')}}
-    "active"
-  {{else if (eq (toLower code) 'suspended')}}
-    "on-hold"
-  {{else}}
-    "{{code}}"
+{{! http://hl7.org/fhir/R4/valueset-medication-statement-status.html }}
+{{#with (toLower code) as |normalizedCode|}}
+  {{#if code}}
```

Comment:
If I understood correctly, we're declaring a new variable `normalizedCode`, right? If so, shouldn't we use it on the `if` as well?

Reply:
good catch - fixed